src/test_queue.py

```diff
@@ -56,62 +55,56 @@
 def test_empty_queue_size():
     """Test to see if any empty queue has length zero."""
     empty_queue = Queue()
-    assert empty_queue.size == 0
+    assert empty_queue.size() == 0
 
 
-def test_empty_queue_pointer():
-    """Test to see if in an empty queue the top is empty."""
-    empty_queue = Queue()
-    assert empty_queue.top is None
-
 # Tests for XL Queue
 
 def test_size_xl():
     """Test init and size on an xl queue."""
-    test_queue = Queue(LONG_queue)
-    assert test_queue.size == 30000
+    test_queue = Queue(LONG_LIST)
+    assert test_queue.size() == 30000
 
 # Tests for Enqueue
 
-@pytest.mark.parametrize('init_value', TYPE_TABLE)
-def test_queue_enqueue_empty(init_value):
+def test_queue_enqueue_empty():
     """Test enqueue on an empty queue."""
     test_queue = Queue()
-    test_queue.enqueue(init_value)
-    assert test_queue.top.value == init_value
+    test_queue.enqueue('test')
+    assert test_queue.peek() == 'test'
 
 
 @pytest.mark.parametrize('init_queue, result', TABLE_LENGTHS)
 def test_enqueue_size(init_queue, result):
     """Test size is correct after a enqueue."""
-    test_queue=Queue(init_value)
+    test_queue = Queue(init_queue)
     test_queue.enqueue('some_string')
-    assert test_queue.size == len(init_queue) + 1
+    assert test_queue.size() == len(init_queue) + 1
 
 # Tests for Dequeue
 
-@pytest.mark.parametrize('init_value', TYPE_TABLE)
-def test_dequeu.init_value):
+
+def test_dequeue():
     """Test dequeue returns correct value."""
-    test_list = DList()
-    test_list.append(init_value)
-    assert test_list.dequeu) == init_value
+    test_list = Queue(['a', 'b', 'c'])
+    assert test_list.dequeue() == 'a'
 
 
-def test_shift_empty():
+def test_dequeue_empty():
     """Test that dequeueing an empty string returns None."""
-    test_list = DList()
-    assert test_list.dequeu) is None
+    test_list = Queue()
+    assert test_list.dequeue() is None
 
 
 @pytest.mark.parametrize('init_list, result', TABLE_LENGTHS)
-def test_shift_length(init_list, result):
+def test_dequeue_length(init_list, result):
     """Test length is correct after a dequeue. If dequeue a zero length list, should still be 0."""
-    test_list = DList(init_list)
-    test_list.dequeu()
-    assert test_list.length == max(0, len(init_list) - 1)
-
-# Tests for Peek
+    test_list = Queue(init_list)
+    test_list.dequeue()
+    assert test_list.size() == max(0, len(init_list) - 1)
 
-????
 
```

Comment:
Jeff, for the test for peek, we just queued a few items up and then made sure that when we peak at an item we got back the first thing we queued.  Let me know if you have any questions about this test.